src/helpers/utils.ts

```diff
@@ -1,18 +1,9 @@
-import assert from "assert";
-import {PUBLIC_KEY_LENGTH, SIGNATURE_LENGTH} from "../constants";
-
-/**
- * Pads byte array with zeroes on left side up to desired length.
- * Throws if source is larger than desired result.
- * @param source
- * @param length
- */
-export function padLeft(source: Uint8Array, length: number): Buffer {
-  assert(source.length <= length, "Given array must be smaller or equal to desired array size");
-  const result = Buffer.alloc(length, 0);
-  result.set(source, length - source.length);
-  return result;
+export function validateBytes(bytesObj: {[argName: string]: Uint8Array | Uint8Array[]}): void {
```

Comment:
```suggestion
export function validateBytes(bytesObj: {[argName: string]: Uint8Array | Uint8Array[]}): asserts bytesObj is NonNullable {
```
Something along these lines, needs some adjustments since we are dealing with object whose values must be notNull.

Maybe `asserts bytesObject[keyOf bytesObject] is NonNullable`

Source: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions

Reply:
But is it really necessary? The argument types do not contain a null type. We are adding this check in case the library is incorrectly used despite the types. Then I'm not sure why the `asserts` instruction is necessary

Reply:
I've implemented your suggestion `asserts bytes is NonNullable` in https://github.com/ChainSafe/bls/pull/31/commits/5db911d4706c532666ec34f08811c2bae192eaa0

Reply:
asserts is for code coverage and typescript typechecking. Just a good practice